corehq/blobs/management/commands/blob_storage_report.py

```diff
@@ -121,7 +121,7 @@ def sumlens(item):
     by_domain = OrderedDict()
     by_type = defaultdict(lambda: defaultdict(list))
     samples = defaultdict(lambda: 0)
-    for i, (domain, domain_sizes) in enumerate(sorted(sizes.items(), key=sumlens)):
+    for i, (domain, domain_sizes) in enumerate(sorted(list(sizes.items()), key=sumlens)):
```

Comment:
this change is not necessary for Python 3 (`sorted` does not need a list as input).

same below